R/data.R

```diff
@@ -0,0 +1,68 @@
+#' Metrics for a single-user simulation suitable for passing to \code{\link{load_runs}}
+#'
+#' A dataset containing latency data for 1323 \code{shinycannon} events.
+#'  The variables are as follows:
+#'
+#' @format A data frame with 1323 rows and 12 variables:
+#' \describe{
```

Comment:
This whole `#' \describe` code block can be moved to `./man-roxygen/slt_demo_columns.R`. (It'd be a short ~15 line file of the copy/pasted roxygen code)

Then this line (and the two identical spots below) would be replaced with `#' @template slt_demo_columns`

Reply:
https://roxygen2.r-lib.org/articles/rd.html#roxygen-templates

Reply:
Ended up using `@eval` instead of `@eval` + templates